scripts/asmCheck_tracer.py

```diff
@@ -31,39 +34,39 @@
 prevPC = ""
 # Strips the newline character
 for line in asmLines:
-  if ">:" in line:
-    asmFunctRetHash[prevPC] = curFunction
-    curFunction = line.strip()
-    firstInst = True
-    continue
-  splitLine = line.strip().split(":")
-  PC = splitLine[0]
-  prevPC = PC
-  if "<" in PC:
-    continue
-  asmHash[PC] = line.strip()
-  if firstInst:
-      asmFunctStartHash[PC] = curFunction
-      firstInst = False
+    if ">:" in line:
+        asmFunctRetHash[prevPC] = curFunction
+        curFunction = line.strip()
+        firstInst = True
+        continue
+    splitLine = line.strip().split(":")
+    PC = splitLine[0]
+    prevPC = PC
+    if "<" in PC:
+        continue
+    asmHash[PC] = line.strip()
+    if firstInst:
+        asmFunctStartHash[PC] = curFunction
+        firstInst = False
 
 printCS = False
-if(args.callStackClk):
+if (args.callStackClk):
     printCS = True
 
 callStack = ["main"]
 
 for line in sstLines:
-  if "Render:" in line and "Core" in line:
-    print(line)
-    clk = line.split("Render:")[1].split(']:')[0].strip()
-    PC = line.split("*I ")[1].split(':')[0].split('0x')[1]
-    so = line.split("*I ")[1].split('\t')[0].strip()
-    if PC in asmHash:
-      o = '{:<60} {:<3} {:<60}'.format(asmHash[PC].strip(),":::", so)
-      if(not args.callStackClk):
-        print(o)
+    if "Render:" in line and "Core" in line:
+        print(line)
+        clk = line.split("Render:")[1].split(']:')[0].strip()
+        PC = line.split("*I ")[1].split(':')[0].split('0x')[1]
+        so = line.split("*I ")[1].split('\t')[0].strip()
+        if PC in asmHash:
+            o = '{:<60} {:<3} {:<60}'.format(asmHash[PC].strip(), ":::", so)
+        if (not args.callStackClk):
```

Comment:
Indentation on this `if` changed, appears to be in a different scope now?

Reply:
> Indentation on this `if` changed, appears to be in a different scope now?

Good catch; I'll check to make sure the nesting is consistent.